onboard-server/src/main/java/no/vegvesen/ixn/serviceprovider/OnboardRestController.java

```diff
@@ -289,13 +292,17 @@ public PrivateChannelApi addPrivateChannel(@PathVariable String serviceProviderN
 			throw new PrivateChannelException("Client channel cannot be null");
 		}
 
-		ServiceProvider serviceProviderToUpdate = getOrCreateServiceProvider(serviceProviderName);
```

Comment:
We still need this one, should be a service provider in the DB on order to setup routing